tools/thuishaven/README.md

```
@@ -38,22 +38,26 @@ Shared, machine-wide (one daemon serves all worktrees):
 
 ```bash
-pnpm dev                 # == haven up: registers hostnames, starts + supervises the stack
+pnpm dev:haven           # == haven up: registers hostnames, starts + supervises the stack
+pnpm dev:single:haven    # …with the workers hosted in-process (one Node process)
 ```
 
 Open <https://langwatch.localhost> to see every stack across your worktrees.
 
 ## Commands
 
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)